src/vogsphere.html

```diff
@@ -15,15 +15,33 @@ <h3>Basics</h3>
             <input id="vogsphere__character-name" name="characterName" type="text" required>
         </div>
 
-        <div class="__input-group">
-            <label for="vogsphere__member-group">Member Group</label>
-            <select id="vogsphere__member-group" name="memberGroup" required>
```

Comment:
The `required` attribute that was previously present is now missing. It should be added to all the new `<input>` fields. (See here for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required)

Reply:
Could I change `required` for `checked` tag? with default input "Crew" ?

Reply:
`required` and `checked` do different things, so they are not interchangeable.

Reply:
Got it! I'll work on it

Reply:
Awesome, thanks!

Reply:
I have added `required` tag to radio selection. But when I was testing it, it is not working the Error message generator so I had to add some JS lines in order to show up that is missing the selection.
`if (!input.memberGroup.value) {`
`            errors.push("ERROR: Missing Member group selection");`
`        }`

Is it ok?